packages/ui-core/src/KitchenSink/KitchenSink.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { KitchenSink } from "./KitchenSink";
+
+// These stories serve as documentation and visual test cases
+// Component Story Format: https://storybook.js.org/docs/react/api/csf
+
+export default {
+  title: "Unsorted/Components/KitchenSink",
+  parameters: { component: KitchenSink }
+};
+
+const Template = args => <KitchenSink {...args} />;
+
+export const Standard = Template.bind({});
+
```

Comment:
Feel free to remove the entire Margaritte export, which will remove the story

Reply:
Does the fact that you're adding the kitchen sink imply that you had everything but the kitchen sink already? :-)

Reply:
@znmeb 😁 I'm pretty sure that's exactly what it means haha